app/components/ProductPage.js

```diff
@@ -1,7 +1,7 @@
-import React from "react";
-import { Link } from "react-router-dom";
-import Button from "react-bootstrap";
-import product from "./Product.js";
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from 'react-bootstrap';
```

Comment:
not really an issue but can always fix it later, import a { Button } or from 'react-bootstrap/Button'